CHANGELOG.md

```diff
@@ -8,7 +8,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
```

Comment:
Think we should add a Changelog entry for these changes. While you are at it you also add one for the integration of the third party Helm charts.

Reply:
i never used integration so i'd prefer to keep it out (as I have no idea how to test it). we can raise another PR for integration thingie. as for the changelog - sure, do you have something in mind?

Reply:
Not sure if you understood me correct here. I just think we are missing a changelog entry for using the third party Helm charts instead of the custom ones we had before.

My proposal:
```
### Added
- Bitnami Helm charts for Kafka, Cassandra and PostgreSQL

### Changed
- extract common credentials into a unified place

### Removed
- custom Helm charts for Kafka, Cassandra and PostgreSQL
```

Reply:
the pr kinda describes all the changes? do you want to put that into the readme or something?

Reply:
No I want to put it into the CHANGELOG.md file so that everyone can see what has changed since the 0.4 release.

Reply:
Think @leonidlm wanted to use the [keepachangelog format](https://keepachangelog.com/en/1.0.0/). Additionally I think all of these changes need to be added to the Unreleased section as 1.0.0 has not yet been released.